languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/remote-rule-filters.xml

```diff
@@ -318,6 +318,26 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 </pattern>
                 <example correction="">Par son absence volontaire de <marker>conjonction</marker> de coordination.</example>
             </rule>
+            <rule>
+                <pattern>
+                    <token regexp="yes">locutions?</token>
+                <marker>
+                    <token regexp="yes">interjective?|adverbiale?</token>
+                </marker>
```

Comment:
the plural forms with `s?`
or with inflected="yes"

Reply:
Other possible words (source: https://fr.wiktionary.org/wiki/locution): 
    locution adjective ou locution adjectivale
    locution adverbiale
    locution conjonctive
    locution déterminative
    locution interjective
    locution nominale
    locution postpositive
    locution prépositive ou locution prépositionnelle
    locution pronominale
    locution verbale
    locution-phrase

Reply:
brilliant!